### PR TITLE
Left side chat

### PR DESCRIPTION
**Pull request by PinkishCow**
Polecat's idea

Left side chat, resize does not work on left but switching back, resizing and then moving it back to the left does work.

I did attempt to get it working for Overrustle, I came to an issue with chrome not allowing the chat embed to affect css in overrustle because of them obviously being different protocols. HTTP vs HTTPS

### CODE REVIEW

File: betterdgg/modules/zleftchat.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
+;(function(bdgg) {
+	//Chat is not resizeable while on the left but can be moved back to the right, resized and moved back
+	//Wait for document to load even though querySelector only returns a value once the page is loaded according to some guy on the internet
+	//$(document).ready(function() {
```
Comment (kenany) [inline betterdgg/modules/zleftchat.js]: So this line isn't necessary?

File: betterdgg/modules/zleftchat.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+					chat = window.parent.document.querySelector("div#chat-panel");
+					bdgg.zleft.leftBoys(bdgg.settings.get('bgg_left_chat'));
+					bdgg.settings.addObserver(function(key, val) {
+						if (key == 'bdgg_left_chat')
+							console.log('changed')
+						bdgg.zleft.leftBoys(val)
+					})
+				},
+				leftBoys: function(val) {
+					if (val == true) {
```
Comment (kenany) [inline betterdgg/modules/zleftchat.js]: This can just be

``` js
if (val) {
```

**Comment by PinkishCow** [comment]
Oh, that's just something i must have forgotten to take out when i was removing all the useless stuff. Sorry

**Comment by downthecrop** [comment]
Nice! If you or Polecat do get that resize fix working we'll try and add that too then.

**Comment by kenany** [comment]
I guess I'll finally dox myself and say that I am SgtMaximum in chat (I better get a flair now Hhhehehe). I feel bad being known only as that random GitHub guy (perhaps I'm not as humble as I thought I was).

**Comment by PinkishCow** [comment]
It now makes sense that you took one look at the issue i was having with CSS and fixed it easily then lol

**Comment by PinkishCow** [comment]
Also I actually looked at the error coming from trying to do left side chat on overrustle and its because iLiedAboutCake decided to embed https but have overrustle http

**Pull request merged**